frontend/src/app/components/tasque-project-settings/project-options-module/issue-template/dropdown-field-edit/dropdown-field-edit.component.html

```diff
@@ -0,0 +1,22 @@
+<div class="dropdown-field">
+    <div class="field-name">
+        <span>{{field.name}}</span>
+    </div>
+    <hr>
+    <div *ngFor="let field of fields" class="dropdown">
+        <span>{{field}}</span>
+        <tasque-button style="margin-left: 10px" (btnClick)="deleteField(field)" [icon]="faMinus" class="btn icon"></tasque-button>
+    </div>
+    <div *ngIf="isChanging" class="edit-dropdown">
+        <tasque-input [formControl]="formControl" class="edit-input" placeholder="Write label name" type="text" (outputValue)="setValue($event)"></tasque-input>
+        <tasque-button class="btn icon" [icon]="faCheck" (keydown.enter)="saveField()" (btnClick)="saveField()"></tasque-button>
+    </div>
+    <div class="create-dropdown">
+        <span>Add new option</span>
+        <tasque-button style="margin-left: 10px" class="btn icon" [icon]="faPlus" (btnClick)="addField()"></tasque-button>
+    </div>
+    <tasque-button style="margin-left: 10px" class="btn mini" text="Finish editing" (btnClick)="finishEdit()"></tasque-button>
+    <div class="data-save">
+        <span>Data will be saved</span>
+    </div>
+</div>
```

Comment:
Forgot a space. Same in a few more files as well

Reply:
Already fixed